src/dir.targets

```diff
@@ -1,18 +1,24 @@
 <Project ToolsVersion="12.0" DefaultTargets="Build" xmlns="http://schemas.microsoft.com/developer/msbuild/2003">
+  
+  <PropertyGroup Condition=" '$(TargetFrameworkIdentifier)' == '' 
+                         and '$(TargetFrameworkVersion)'    == '' 
+                         and '$(TargetFrameworkProfile)'    == '' ">
+    <TargetingDefaultPlatform>true</TargetingDefaultPlatform>
+  </PropertyGroup>
 
-  <PropertyGroup Condition="'$(TargetFrameworkVersion)' != '' and '$(TargetFrameworkProfile)' != ''">
-    <TargetingPortable>true</TargetingPortable>
+  <PropertyGroup Condition="'$(TargetFrameworkIdentifier)' == ''">
+     <TargetFrameworkIdentifier>.NETPortable</TargetFrameworkIdentifier>
   </PropertyGroup>
```

Comment:
Do you need to set this here? I guess the Immutable/Metadata/DataFlow things that are using the default portable currently instead of the package resolve are the only ones that would need this correct?